features/admin.branding.v1/components/screen-dropdown.tsx

```diff
@@ -66,6 +70,9 @@ const ScreenDropdown: FunctionComponent<ScreenDropdownPropsInterface> = (
         onChange(defaultScreen);
     }, [ defaultScreen ]);
 
+    const disabledFeatures: string[] = useSelector((state: AppState) =>
+        state?.config?.ui?.features?.branding?.disabledFeatures) || [];
```

Comment:
Shall we rename the variable to be more specific?
Ex: `disabledBrandingFeatures`

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7005/commits/d71e795cadeda58c790fa54f08ada0058b336c18